src/Razor/test/testapps/Directory.Build.targets

```diff
@@ -2,7 +2,11 @@
   <Import Project="RazorTest.Introspection.targets" />
 
   <PropertyGroup>
-    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == '$(DefaultNetCoreTargetFramework)' ">$(MicrosoftNETCoreApp31PackageVersion)</RuntimeFrameworkVersion>
+    <!--
+      The runtime version is only useful to restore and compile test projects, and not used to run tests.
+      We'll keep it pinned to a version that has already been shipped.
+     -->
+    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == '$(DefaultNetCoreTargetFramework)' ">3.1.0</RuntimeFrameworkVersion>
```

Comment:
Suggest changing this to `$(MicrosoftNETCoreAppRefPackageVersion)`

Reply:
```suggestion
    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == '$(DefaultNetCoreTargetFramework)' ">$(MicrosoftNETCoreAppRefPackageVersion)</RuntimeFrameworkVersion>
```